### ISSUE TITLE
add view.show_only?

### ISSUE THREAD

**Issue by hainm** [arose/nglview#207]
currently I need to view 61 trajectories (wt protein with different mutants)

```python
view.show_only([1, 3])  # show only component 1 and 3
```

or just
```python
view.show([1, 3])
```

**Comment by hainm** [comment]
```python
view.hide_all_but(...)
```

LOL

**Comment by hainm** [comment]
closed by #208

**Comment by hainm** [comment]
notebook: http://nbviewer.jupyter.org/github/arose/nglview/blob/master/nglview/tests/notebooks/show_only.ipynb
